src/main/java/nl/fuchsia/controller/ZaakController.java

```diff
@@ -15,18 +16,23 @@
 
     private final ZaakService zaakService;
 
+    @Autowired
     public ZaakController(ZaakService zaakService) {
         this.zaakService = zaakService;
     }
 
     @PostMapping
-    public void addZaak(@Valid @RequestBody Zaak newZaak) {
-        zaakService.addZaak(newZaak);
+    public ResponseEntity<Zaak> addZaak(@Valid @RequestBody Zaak zaak) {
+        return ResponseEntity.ok(zaakService.addZaak(zaak));
     }
 
     @GetMapping
-    // Is niet nodig voor Story, toegevoegd om te kijken of zaak daadwerkelijk in List is geplaatst.
     public ResponseEntity<List> getZaken() {
         return ResponseEntity.ok().body(zaakService.getZaken());
     }
+
```

Comment:
Hier misschien wel comment bij plaatsen, zit in andere story

Reply:
Gaat om methode zaakById. Zit inderdaad nergens in een story. We laten deze staan.